.github/workflows/e2e-leg-4.yml

```
@@ -15,6 +15,9 @@ on:
       vertica-deployment-method:
         type: string
         required: false
+      vertica-superuser-name:
+        type: string
+        required: false
```

Should we set the default?
```suggestion
        required: false
        default: "myadmin"
```

---

I set the default value in workflow_dispatch.